packages/amplify-util-mock/src/utils/lambda/populate-cfn-params.ts

```diff
@@ -61,9 +61,15 @@ const getAmplifyMetaParams = (
         );
         print.warning('This attribute will be undefined in the mock environment until you run `amplify push`');
       }
-      if (overrideApiToLocal && attribute === 'GraphQLAPIEndpointOutput') {
-        val = `http://localhost:${MOCK_API_PORT}/graphql`;
+
+      if (overrideApiToLocal) {
+        if (attribute === 'GraphQLAPIEndpointOutput') {
+          val = `http://localhost:${MOCK_API_PORT}/graphql`;
+        } else if (attribute === 'GraphQLApiKeyOutput') {
```

Comment:
I think this should be
```suggestion
        } else if (attribute === 'GraphQLAPIKeyOutput') {
```